tests/unit/test_install_python_libraries.py

```diff
@@ -150,9 +152,12 @@ def test_install_libraries_when_no_splunktaucclib_is_present_but_has_ui(tmp_path
     tmp_lib_reqs_file = tmp_lib_path / "requirements.txt"
     tmp_lib_reqs_file.write_text("solnlib\nsplunk-sdk\n")
 
+    lib = list(LIBS_REQUIRED_FOR_UI)[0]
+    ver = LIBS_REQUIRED_FOR_UI[lib]
+
     expected_msg = (
-        f"splunktaucclib is not found in {tmp_lib_reqs_file}. "
-        f"Please add it there because this add-on has UI."
+        f"This add-on has an UI, so the {lib} is required but not found in {tmp_lib_reqs_file}. "
+        f"Please add it there and make sure it is at least version {ver}."
     )
```

Comment:
For the sake of simplicity, we can just say `splunktaucclib` here and the specific version.

Reply:
I made it this way for easier maintenance because we use this in two tests already but sure, I can change that